xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/resources/liveTableLiveDataConfiguration.json

```diff
@@ -176,7 +176,8 @@
     ],
 
     "displayers": [
-      {"id": "link", "propertyHref": "doc.url"}
+      {"id": "link", "propertyHref": "doc.url"},
+      {"id": "docTitle", "propertyHref": "doc.url"}
```

Comment:
Here is where I would list the ``docTitle`` and ``xObjectProperty`` displayers (not in the live data API).